public/application/bootstrap/autoload.php

```diff
@@ -7,3 +7,15 @@
 # Add the vendor directory to the include path
 ini_set('include_path', dirname(__DIR__, 3) . "/vendor" . PATH_SEPARATOR . get_include_path());
 
+# Include the environment variables
+try {
+    // Include the env
+    $dotenv = new Dotenv\Dotenv(dirname(__DIR__, 3));
+    $dotenv->load();
+
+    // Include the deploy number
+    $dotenv = new Dotenv\Dotenv(dirname(__DIR__, 3), '.deploy');
```

Comment:
If the first one doesn't work I'm fine with giving up on loading the second one.